README.md

```diff
@@ -78,3 +78,11 @@ This is a way to incrementally build a version over time and only release a vers
 | Default      | Description                                       |
 | ------------ | ------------------------------------------------- |
 | `dependabot` | The label to indicate that the action should run. |
+
+#### `entryPrefix`
+
+| Default      | Description                                       |
+| ------------ | ------------------------------------------------- |
+| `Bump`       | The start of a dependency bump entry line. |
+
+The format of an entry is `- <entryPrefix> <package> from <oldVersion> to <newVersion>`. If a previous entry was written with a different entry (`Bump` vs `Bumps`), the entry will still get updated for updates within the same version.
```

Comment:
I just realized that the existing code will fail if a multi-word prefix is provided (the regexes assume they are looking for a single word). We should document and enforce the single-word constraint. Or implement support for multi-word, but that might not be worth the complexity. What do you think?

Reply:
I think keeping it to a single word is good for now. If it becomes a need from other consumers, we are can always add it. 

I'll update the docs here to reflect that